packages/coinstac-ui/app/render/index.js

```diff
@@ -28,10 +27,9 @@ app.analysisRequestId = app.analysisRequestId || 0;
 require('./styles/app.scss');
 
 const rootEl = document.getElementById('app');
-const store = configureStore();
 
 render(
-  <Root history={hashHistory} store={store} />,
```

Comment:
Good point. I was looking for a way to get around the `PropTypes.object` error early on, before you advised me to cancel that rule, and put this in place before immediately forgetting I had. I've reverted back to the way it was before.